### PR TITLE
Add ability to hide system metrics from table

### PR DESCRIPTION
**Pull request by arsengit**
- Added `Hide System Metrics` button in `Manage Columns` popover 
- Added ability to hide system metrics from explorer tables
- Added `xSmall` size for `Button` component

### CODE REVIEW

**Comment by arsengit** [comment]
Resolves #1198

**Review by VkoHov — APPROVED** [review]
lgtm!

File: aim/web/ui/src/services/models/explorer/createAppModel.ts [hunk 1]
```diff
@@ -340,6 +340,7 @@ function createAppModel(appConfig: IAppInitialConfig) {
           config.table = {
             resizeMode: ResizeModeEnum.Resizable,
             rowHeight: RowHeightSize.md,
+            hideSystemMetrics: true,
```
Comment (KaroMourad) [inline aim/web/ui/src/services/models/explorer/createAppModel.ts:343]: let's add this config state also for Metrics table

File: aim/web/ui/src/services/models/explorer/createAppModel.ts [hunk 2]
```diff
@@ -2294,6 +2282,28 @@ function createAppModel(appConfig: IAppInitialConfig) {
...
+          sameValueColumns: string[];
+        },
+        tableColumns: ITableColumn[],
+        configData: IAppModelConfig | any,
+      ): void {
+        if (updateTableTimeoutId) {
+          clearTimeout(updateTableTimeoutId);
+        }
+
+        updateTableTimeoutId = setTimeout(() => {
```
Comment (KaroMourad) [inline aim/web/ui/src/services/models/explorer/createAppModel.ts:2297]: here let's use `window.setTimeout` and fix the type of `updateTableTimeoutId: ReturnType<typeof setTimeout>;`
to number

File: aim/web/ui/src/services/models/explorer/createAppModel.ts (hunk 1 above)
Comment (arsengit) [reply aim/web/ui/src/services/models/explorer/createAppModel.ts:343]: @KaroMourad Metrics table haven't system metrics

File: aim/web/ui/src/services/models/explorer/createAppModel.ts (hunk 2 above)
Comment (arsengit) [reply aim/web/ui/src/services/models/explorer/createAppModel.ts:2297]: @KaroMourad  There are no difference, except that windows.setTimeout works only in browser environment

File: aim/web/ui/src/services/models/explorer/createAppModel.ts (hunk 1 above)
Comment (arsengit) [reply aim/web/ui/src/services/models/explorer/createAppModel.ts:343]: In Metrics table the Hide/Show button isn't rendered

Comment (KaroMourad) [reply aim/web/ui/src/services/models/explorer/createAppModel.ts:343]: @arsengit the thing is we can have the same table config for all of them to easily integrate for every app model, we need it for app model unification @roubkar  what do you think ?

File: aim/web/ui/src/services/models/explorer/createAppModel.ts (hunk 2 above)
Comment (KaroMourad) [reply aim/web/ui/src/services/models/explorer/createAppModel.ts:2297]: @arsengit we can use it only on browser env that's why we can use `window.setTimeout` here

File: aim/web/ui/src/services/models/explorer/createAppModel.ts (hunk 1 above)
Comment (roubkar) [reply aim/web/ui/src/services/models/explorer/createAppModel.ts:344]: we can think of this during base explorer implementation, for now this is ok I think

**Pull request merged**